scripts/build.js

```diff
@@ -75,7 +61,7 @@ async function runBuild(entry, outfile) {
       svelteCompilePlugin,
       TsconfigPathsPlugin({ tsconfig: tsconfigPathPluginContent }),
       nodeExternalsPlugin({
-        allowList: ["@budibase/frontend-core", "svelte"],
+        allowList: ["@budibase/frontend-core", "@budibase/pro", "svelte"],
```

Comment:
Pro will be now bundled, from the source code with the submodule loaded or from node_modules for OSS users